server/controllers/projects.controller.js

```diff
@@ -0,0 +1,78 @@
+const Projects = require("../models/projects.model");
+const _myCounter = require("../services/_counter.service");
+const createError = require("http-errors");
+const Tasks = require("../models/tasks.model");
+
+const getManyProjects = async (req, res, next) => {
```

Comment:
We should not work with DB inside controller, please move this part to some service. Same for all functions in all controllers